src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,62 @@
-export const Pagination = () => {};
+type Props = {
+  pages: string[];
+  pageCur: number;
+  setPage: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({ pages, pageCur, setPage }) => {
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${pageCur !== 1 ? '' : 'disabled'}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={pageCur === 1 ? 'true' : 'false'}
+          onClick={() => {
+            if (pageCur !== 1) {
+              setPage(pageCur - 1);
+            }
+          }}
+        >
+          «
+        </a>
+      </li>
+      {
+        pages.map(page => {
+          return (
+            <li className={`page-item ${+page === pageCur ? 'active' : ''}`}>
```

Comment:
The 'li' elements created by the map function need a unique 'key' prop to avoid potential rendering issues.